ballista/rust/core/proto/ballista.proto

```diff
@@ -732,6 +740,14 @@ message LaunchTaskResult {
   // TODO when part of the task set are scheduled successfully
 }
 
+message CancelTasksParams {
+  repeated PartitionId partition_id = 1;
+}
+
```

Comment:
Can we name the partition_id to task_id here ?

````
  PartitionId task_id = 1;

Reply:
I'd like to get this PR merged. Feel free to create a follow-on issue or PR to discuss this and other naming issues.